hotham/src/systems/stage.rs

```diff
@@ -23,20 +21,3 @@ pub fn add_stage(world: &mut World, physics_context: &mut PhysicsContext) -> Ent
         rigid_body,
     ))
 }
-
-/// Update player's views to take into account the current position of the Stage in global space
-///
-/// Must happen each tick after parent transforms have been updated.
-pub fn update_views_with_stage_transform(world: &mut World, views: &mut [xr::View]) {
```

Comment:
This function should not be called anymore because the views are already in the correct coordinate system.